### ISSUE TITLE
RecycleableFuelsRods exploit with proliferator

### ISSUE THREAD

**Issue by Magiclaze** [jinxOAO/DSPmods_BepInEx#50]
Extra products is not disabled when you refuel your empty fuel rod or when making them, doubling the extra products chance by 2. Maybe deactivating extra prodcuts on the refuelling could feel less cheaty since you already conserve the fuel rod and don't need to duplicate it.

**Comment by jinxOAO** [comment]
That's right, and I prefer to disable the extra product when makeing empty rod. because some players reported that they sometimes found the fuel rods would disappear. so the extra product might help them make up for the inexplicable disappearance. to be honest, I have not found the reason. Anyway, I will consider disable the refuel or/and empty rod's extra product in next version.
